src/providers/actions-service/actions-service.ts

```diff
@@ -13,7 +13,7 @@ interface FetchActionsParams {
   offset?: string;
   limit?: string;
   include?: string | string[];
-  [key: string]: string | string[];
+  [ key: string ]: string | string[];
```

Comment:
```suggestion
  [key: string]: string | string[];
```

This is an index signature, not an array.